### ISSUE TITLE
UI to change color scheme

### ISSUE THREAD

**Issue by Sternbach-Software** [NUmeroAndDev/MaterialGallery-android#570]
I really wish there was a UI in app where I could change the color scheme for the material components catalog and see how it looks on all of the material components. I am trying to show my boss the different color schemes I came up with (I already have the hex values) and want to show him what the app might look like, based on the individual components. I was seriously considering making my own app because none of the tens of apps I looked through had this.

**Comment by sanjaykumarm** [comment]
I am agree with @shmueldabomb441. It should be there and it will be a great enhancement to the app. It will also help to others who want to give custom theme option within the app apart from light, dark theme option.

**Comment by shmueldabomb441** [comment]
The challenge is that the entire app is built based on predefined colors in the styles.xml, so it would be a very big rework of complicated code.

**Comment by sanjaykumarm** [comment]
Yes I can understand that but if someone get some free time add that then it would be a great enhancement. Again I am little confused with the material3 theme branding like _**user defined color scheme**_ and **_dynamic theme_**, so in material3 didn't they make any provision to make it easy ? Or does that only mean - defining color schemes based on primary brand color ?
